Windows: startup_mode: Maximized no longer works

System: Windows 10, Alacritty 0.3.3.

Using `startup_mode: Maximized` has the same effect as `startup_mode: Windowed`.
However, `Fullscreen` works fine.

This seems to be a regression compared to before.

@chrisduerr Your patch had some issues,
```
[alacritty]$ git apply fullscreen.patch
[alacritty]$ cargo build --release
   Compiling alacritty_terminal v0.3.3 (C:\cygwin64\usr\local\src\alacritty\alacritty_terminal)
error: unexpected close delimiter: `}`
   --> alacritty_terminal\src\display.rs:596:1
```

There was an extra ")" character from "any" that wasn't removed, I removed the extra ")" and got another compilation error:
```
error[E0599]: no method named `is_x11` found for type `window::Window` in the current scope
   --> alacritty_terminal\src\display.rs:249:46
    |
249 |             StartupMode::Maximized if window.is_x11() => window.set_maximized(true),
    |                                              ^^^^^^
    |
   ::: alacritty_terminal\src\window.rs:59:1
    |
59  | pub struct Window {
    | ----------------- method `is_x11` not found for this
```

I removed `if window.is_x11()`, it compiled, but `startup_mode: Maximized` still behaves the same.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -41,6 +41,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Freeze when application is invisible on Wayland
 - Emoji colors blending with terminal background
 - Fix escapes prematurely terminated by terminators in unicode glyphs
+- Incorrect location when clicking inside an unfocused window on macOS
+- Startup mode `Maximized` on Windows
 
 ## 0.4.2
 

--- a/font/Cargo.toml
+++ b/font/Cargo.toml
@@ -17,10 +17,10 @@ servo-fontconfig = "0.5.0"
 freetype-rs = "0.25"
 
 [target.'cfg(target_os = "macos")'.dependencies]
-core-foundation = "0.6"
-core-text = "13"
-core-graphics = "0.17"
-core-foundation-sys = "0.6"
+core-foundation = "0.7"
+core-text = "15"
+core-graphics = "0.19"
+core-foundation-sys = "0.7"
 
 [target.'cfg(windows)'.dependencies]
 dwrote = { version = "0.11" }

